docs/guides/build-xrd.md

```diff
@@ -0,0 +1,202 @@
+---
+tocHidden: true
+---
+
+# Build your XRD

```

Comment:
Instead of a top level header `#` make this the `title:` value in the front matter.